brainstormforce/ultimate-addons-for-gutenberg
# Fix: Testimonial - Not visible in editor.

### Description

- Updated slick CSS for testimonial block.

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop testimonial block.
- Check Is it visible in the editor?
- Test by changing setting option of testimonial
- Drag and drop post carousel check is there any conflict between 2 block.

### Checklist:
